<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TMEdit</class>
 <widget class="QDialog" name="TMEdit">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>571</width>
    <height>601</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QToolButton" name="tbLeft">
       <property name="text">
        <string>Up</string>
       </property>
       <property name="icon">
        <iconset resource="resrs.qrc">
         <normaloff>:/images/go-up.png</normaloff>:/images/go-up.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MLabel" name="tileShow">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>32</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>CT</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbRight">
       <property name="text">
        <string>Down</string>
       </property>
       <property name="icon">
        <iconset resource="resrs.qrc">
         <normaloff>:/images/go-down.png</normaloff>:/images/go-down.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="l2box">
       <property name="text">
        <string>L2</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="x2box">
       <property name="text">
        <string>x2</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QToolButton" name="tbOpen">
       <property name="text">
        <string>O</string>
       </property>
       <property name="icon">
        <iconset resource="resrs.qrc">
         <normaloff>:/images/document-open.png</normaloff>:/images/document-open.png</iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="tbSave">
       <property name="text">
        <string>S</string>
       </property>
       <property name="icon">
        <iconset resource="resrs.qrc">
         <normaloff>:/images/document-save.png</normaloff>:/images/document-save.png</iconset>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="MLabel" name="tileLine">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>64</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>512</width>
         <height>64</height>
        </size>
       </property>
       <property name="text">
        <string>TileRow</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="MLabel" name="tilemap">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>512</width>
         <height>512</height>
        </size>
       </property>
       <property name="text">
        <string>Tilemap</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MLabel</class>
   <extends>QLabel</extends>
   <header location="global">mlabel.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resrs.qrc"/>
 </resources>
 <connections/>
</ui>
